armotypes/applicationprofiletypes.go

```diff
@@ -0,0 +1,14 @@
+package armotypes
+
+type UserApplicationProfileRequest struct {
```

Comment:
you should use this structure
https://github.com/armosec/armoapi-go/blob/dabb4122ca5ed6bbdcfe51f95e844be513fc02c3/armotypes/portaltypes.go#L49